lld/wasm/Config.h

```diff
@@ -141,6 +142,9 @@ struct Ctx {
   llvm::SmallVector<InputGlobal *, 0> syntheticGlobals;
   llvm::SmallVector<InputTable *, 0> syntheticTables;
 
+  // Linker-generated symbols like __wasm_init_memory, __heap_base, etc.
+  std::unique_ptr<WasmSym> wasmSym;
```

Comment:
Why can't this just be normal struct member?  (i.e. why does need to be a pointer at all).

Reply:
It looks like the ELF linker calls this member just `sym` .. can we do the same there?